src/libstore/unix/build/derivation-builder.cc

```diff
@@ -506,11 +506,6 @@ std::variant<std::pair<BuildResult::Status, Error>, SingleDrvOutputs> Derivation
            being valid. */
         auto builtOutputs = registerOutputs();
 
-        StorePathSet outputPaths;
-        for (auto & [_, output] : builtOutputs)
-            outputPaths.insert(output.outPath);
-        runPostBuildHook(store, *logger, drvPath, outputPaths);
-
         /* Delete unused redirected outputs (when doing hash rewriting). */
         for (auto & i : redirectedOutputs)
             deletePath(store.Store::toRealPath(i.second));
```

Comment:
This means the post build hook can't do anything with these paths, but I think that's good. (And it would probably have hard time finding them anyways.)